cmd/reconciler/ip.go

```diff
@@ -4,13 +4,48 @@ import (
 	"context"
 	"flag"
 	"os"
+	"strings"
 
 	"github.com/k8snetworkplumbingwg/whereabouts/pkg/logging"
 	"github.com/k8snetworkplumbingwg/whereabouts/pkg/reconciler"
 )
 
 const defaultReconcilerTimeout = 30
 
+// Matches known error cases from which the process should exit cleanly.
+func knownErrorCase(err error) bool {
```

Comment:
It may be interesting to actually store this out around in some var. Like, setting a threshold of where this actually may be hitting a real issue sounds reasonable to me. Like a timeout may be on the infra, multiple ones may not be.

Reply:
Hey Daniel, that's a nice suggestion. What I've done is updated this to cycle through a constant set of known errors for now, and potentially in a future edition make this configurable. Thanks!